packages/web-components/fast-foundation/src/horizontal-scroll/horizontal-scroll.ts

```diff
@@ -140,7 +140,7 @@ export class HorizontalScroll extends FoundationElement {
      * Attribute to hide flippers from assistive technology
      * @public
      */
-    @attr({ attribute: "aria-hidden", converter: booleanConverter })
+    @attr({ attribute: "flippers-hidden-from-at", converter: booleanConverter })
     public flippersHiddenFromAT: boolean = false;
```

Comment:
Won't this be a breaking change for anyone already using the `aria-hidden` attribute to set this property?

Reply:
I think this is fundamentally broken - so I’m not sure we should continue supporting this as it removes the entire control from the DOM. It’s breaking but I think that’s the expected fix as this is broken.

Reply:
Similarly, adding aria-hidden to the control would continue to work as expected in that case - I think?